.gitignore

```diff
@@ -0,0 +1,2 @@
+/node_modules
```

Comment:
- [ ] Please add `node_modules` instead of `/node_modules` in the `.gitignore` file. Currently, you have the node_modules folder on your repo. Please remove it from here.